agent/src/iconfucius/client/server.py

```
@@ -936,24 +949,48 @@ def log_message(self, fmt, *args):
+def run_server(port: int = 55129, open_browser: bool = True):
+    """Start the UI server.
+
+    Called by ``iconfucius ui`` or ``python -m iconfucius.client.server``.
+    """
+    try:
+        server = ThreadingHTTPServer(("127.0.0.1", port), UIHandler)
+    except OSError as exc:
+        if "Address already in use" in str(exc):
+            project = os.environ.get("ICONFUCIUS_ROOT", os.getcwd())
+            print(f"\nPort {port} is already in use.")
+            print(f"Another iconfucius ui is probably running.\n")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)